Authoring/AuthoringSample/AuthoringSample.csproj

```diff
@@ -0,0 +1,18 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
+    <LangVersion>preview</LangVersion>
+    <AssemblyVersion>1.0.0.0</AssemblyVersion>
+    <EnableCsWinRTLogging>true</EnableCsWinRTLogging>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <CompilerVisibleProperty Include="AssemblyName" />

```

Comment:
I assume AssemblyName has a reasonable default value?

Reply:
Yes in this sample the default seems to be AuthoringSample.